app/views/external/landing.html.haml

```diff
@@ -20,12 +20,20 @@
 #top-menu.top-menu= render "layouts/menu"
 
 .page-box
+  -if FALSE
+    .container
+      %h4 Latest announcements
+      %li
+        <a href="https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource/" title="Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG">Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG</a> (September 2, 2020)
+    %br/
+    %br/
```

Comment:
Don't add breaking spaces in HTML, instead use margin bottom on elements you want to have a space after. In this case you can add `.mb-3` on the container class to get the same effect.

```
.container.mb-3
  %h4 ...
  %li
```

Reply:
Done